compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/SwingPanel.desktop.kt

```diff
@@ -201,12 +166,11 @@ private class SwingInteropViewGroup(
     }
 }
 
-private class FocusSwitcher(
-    private val interopViewHolder: InteropViewHolder,
+internal class FocusSwitcher(
```

Comment:
Moved

Reply:
Since it's not file-private anymore, probably the name should be a bit more specific. Like `InteropFocusSwitcher`? WDYT about moving it into `viewinterop` package?

Reply:
Yes, rename sounds good. Don't want to move anything from `awt` for now.

Reply:
Deprecations is a different topic, I don't think it's related to internal APIs